StreamChatClient_v3/Utils/Logger/Logger.swift

```diff
@@ -0,0 +1,212 @@
+//
+// Logger.swift
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+public var log: Logger {
```

Comment:
nit: Missing comment

Reply:
On a second thought, shouldn't this be `internal`? We can have internal loggers for all our SDKs, doesn't make sense to have a single public one